rollup.config.js

```diff
@@ -0,0 +1,24 @@
+import ts from '@rollup/plugin-typescript'
+import { terser } from '@rollup/plugin-terser'
+
+export default {
+  input: 'src/main/minio.ts',
+  output: [
+    {
+      file: 'dist/main/minio.cjs',
+      format: 'cjs',
+      exports: 'default'
+    },
+    {
+      file: 'dist/main/minio.mjs',
+      format: 'es'
+    },
+    {
+      file: 'dist/main/minio.iife.js',
+      format: 'iife',
+      name: 'MinIO',
+      plugins: [terser()]
```

Comment:
IIFE output is actually optional, as I believe this will be a broken build once used.